.travis.yml

```diff
@@ -13,8 +13,9 @@ script:
   - make test
   - bundle exec codeclimate-test-reporter
 env:
-  - RACK_ENV=test
-  - RAILS_ENV=test
+- RACK_ENV=test
+- RAILS_ENV=test
+- COVERAGE=true
```

Comment:
OMG we were running one build per env var before

> When you define multiple variables per line in the env array (matrix variables), one build is triggered per item.

https://docs.travis-ci.com/user/environment-variables/